node/src/tasks/bsp_upload_file.rs

```diff
@@ -328,12 +328,12 @@ where
     async fn on_file_complete(&self, file_key: &HasherOutT<T>) {
         info!(target: LOG_TARGET, "File upload complete ({:?})", file_key);
 
-        // Unregister the file from the file transfer service.
-        self.storage_hub_handler
-            .file_transfer
-            .unregister_file(file_key.as_ref().into())
-            .await
-            .expect("File is not registered. This should not happen!");
```

Comment:
Why did you comment this?

Reply:
It was throwing an error and I wanted it to just get to the end. We can fix it in another PR, but I would want to merge this for Tim.